src/routes/choice-exercises/__tests__/ArticleChoiceExerciseScreen.spec.tsx

```diff
@@ -57,29 +62,52 @@ describe('ArticleChoiceExerciseScreen', () => {
   it('should allow to skip an exercise and try it out later', () => {
     mockUseLoadFromEndpointWitData(testDocuments)
 
-    const { getByText } = render(<ArticleChoiceExerciseScreen route={route} navigation={navigation} />, {
+    const { getByText, getAllByText } = render(<ArticleChoiceExerciseScreen route={route} navigation={navigation} />, {
       wrapper: wrapWithTheme
     })
 
+    expect(getAllByText('Helm')[0]).toBeDefined()
```

Comment:
Why do you use `getAllByText` here and assert just on the first element anyway? Same below

Reply:
Answer solutions for ArticleChoice exercises are:
Der Helm
Die Helm
Das Helm
Die Helm (Plural)
To check if the correct word, is asked/prompted it is enough to just check the first is the word we are looking for, all are the same but getByText fails if the text exists more than one time.

Reply:
Then I think you should assert `expect(getAllByText('Helm')).toHaveLength(4)`

Reply:
changed it, but as this is not what the test tests i think it really does not matter.

Reply:
Why is the length `3`?

Reply:
Because `Helm (Plural)` does not match `Helm`. getByText is always a full-match.

Reply:
Then please use `getAllByText(/Helm/)`